src/sentry/static/sentry/app/components/emptyStateWarning.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'react-emotion';
+// import {Panel, PanelBody} from './panels';
+
+class EmptyStateWarning extends React.Component {
```

Comment:
Can we extend `<EmptyMessage>` here?

Reply:
Apart from their purpose there's not a lot that's actually the same, would rather keep 2 simple components than make one complicated one

Reply:
It might be worthwhile to eventually align the design down the line though